app/views/concerto_plugins/_plugins_navigation.html.erb

```diff
@@ -7,7 +7,7 @@
     </li>
     <li class="active">
       <%= link_to concerto_plugins_path do %>
-          <i class="icon-bolt"></i> <%= t :manage_model, :model => ConcertoPlugin.model_name.human.pluralize %>
+          <i class="icon-bolt"></i> <%= t :manage_model, :model => ConcertoPlugin.model_name.human(count: 42) %>
```

Comment:
this is a little better, but still always the same (probably always other: in the translation count), could we use ConcertoPlugin.count)